e2e-test/log/README.md

```diff
@@ -13,6 +13,19 @@ or from project root directory
 ./gradlew :e2e-test:log:basicLogEndToEndTest
 ```
 
+## Basic Grok Ingestion Pipeline with AWS secrets End-to-end test
+
+This test requires AWS secrets `opensearch-sink-basic-credentials` with the basic credentials for opensearch sink to be created in `us-east-1` within the test AWS account. The IAM credentials needs to be passed in as environment variables.
+
+Run from current directory
+```
+AWS_ACCESS_KEY_ID=<your-aws-access-key-id> AWS_SECRET_ACCESS_KEY=<your-aws-secret-key> AWS_SESSION_TOKEN=<your-aws-session-token> ./gradlew :basicLogWithAwsSecretsEndToEndTest
```

Comment:
I think we'd be better off supporting `AWS_PROFILE` instead. Then it is up to an individual developer how to configure a profile. It could use a role, or could be the secret keys directly.

Reply:
Attempted but got the following error when trying to mount ~/.aws:

```
 Status 400: {"message":"create ~/.aws: \"~/.aws\" includes invalid characters for a local volume name, only \"[a-zA-Z0-9][a-zA-Z0-9_.-]\" are allowed. If you intended to pass a host directory, use absolute path"}
```
It might be a restriction with gradle docker library

Reply:
resolved